webpack/src/Extends/Host/PuppetTab/Routes.js

```diff
@@ -21,7 +22,7 @@ const SecondaryTabRoutes = ({ hostName, hostInfo, status }) => (
       <EmptyPage header="Smart class parameters" />
     </Route>
     <Route path={route('yaml')}>
-      <EmptyPage header="YAML" />
+      {hostName ? <YamlTab hostName={hostName} /> : <EmptyPage header="YAML" />}
```

Comment:
question to @marisvirik, did we rename the tab to `ENC` instead of `YAML`?

Reply:
For reports, we rename it ENC or ENC preview if that makes more sense. @Ron-Lavi

Reply:
@nadjaheitmann @ares what are your thoughts? :)

Reply:
It is called `YAML` on the old host page, in the current it is `Puppet YAML`. I prefer `Puppet YAML` over `YAML`, but I don't have a strong preference on whether this should be `ENC` or not. @ares might be a better person to ask.

Reply:
@ares is out of office for next two days. ENC or ENC preview was his suggestion.

Reply:
`ENC preview` sounds great 👍🏼